modules/oe_list_pages_open_vocabularies/tests/src/Kernel/ListPagesSearchApiConfiguratorTranslationTest.php

```diff
@@ -0,0 +1,135 @@
+    $this->assertTrue($config_translation->isNew());
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'facets.facet.' . $facet_id_two);
+    $this->assertTrue($config_translation->isNew());
+
+    // Create a translation of the association.
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'open_vocabularies.open_vocabulary_association.' . $association->id());
+    $config_translation->set('label', 'New association label in French');
+    $config_translation->save();
+
+    // Assert that the facets have also been translated.
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'facets.facet.' . $facet_id_one);
+    $this->assertFalse($config_translation->isNew());
+    $this->assertEquals('New association label in French', $config_translation->get('name'));
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'facets.facet.' . $facet_id_two);
+    $this->assertFalse($config_translation->isNew());
+    $this->assertEquals('New association label in French', $config_translation->get('name'));
+
+    // Update the translation and assert the changes are reflected.
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'open_vocabularies.open_vocabulary_association.' . $association->id());
+    $config_translation->set('label', 'New association label in French updated');
+    $config_translation->save();
+
+    /** @var \Drupal\language\Config\LanguageConfigOverride $config_translation */
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'facets.facet.' . $facet_id_one);
+    $this->assertFalse($config_translation->isNew());
+    $this->assertEquals('New association label in French updated', $config_translation->get('name'));
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'facets.facet.' . $facet_id_two);
+    $this->assertFalse($config_translation->isNew());
+    $this->assertEquals('New association label in French updated', $config_translation->get('name'));
+
+    // Delete the translation and assert the facet translation is also
+    // deleted.
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'open_vocabularies.open_vocabulary_association.' . $association->id());
+    $config_translation->delete();
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'facets.facet.' . $facet_id_one);
+    $this->assertTrue($config_translation->isNew());
+    $this->assertEmpty($config_translation->get());
+    $config_translation = \Drupal::languageManager()->getLanguageConfigOverride('fr', 'facets.facet.' . $facet_id_two);
+    $this->assertTrue($config_translation->isNew());
+    $this->assertEmpty($config_translation->get());
```

Comment:
Did you mean get 'name'?
```suggestion
    $this->assertEmpty($config_translation->get());
```